.github/CODEOWNERS

```diff
@@ -188,6 +188,7 @@ go_deps.bzl               @dfinity/idx
 /rs/query_stats/                                        @dfinity/execution @dfinity/consensus
 /rs/recovery/                                           @dfinity/consensus
 /rs/registry/                                           @dfinity/nns-team
+/rs/registry/node_provider_rewards                      @dfinity/dre
```

Comment:
@aterga Did you mean to only specify the DRE team?

Reply:
Yes, my reasoning was that this crate contains only business logic that would require DRE's approval and not require NNS team's approval. Wdyt?

Reply:
We have a plan to move in that direction after we move the rewards calculations into a DRE-owned canister, but at this point, since it runs in Registry, we should own it.

Reply:
```suggestion
```